src/20_Components/amp-live-list.html

```diff
@@ -0,0 +1,147 @@
+  
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <style amp-custom>
+    amp-live-list > [items] {
+      display: flex;
+      flex-direction: column;
+    }
+
+    .amp-live-list-item {
+      background-color: #f8f8f8;
+      border-bottom: 1px solid rgba(0, 0, 0, .12);
+    }
+
+    .card-list {
+      display: flex;
+      margin: 15px;
+      padding: 10px;
+      align-items: center;
+      background-color: #fff;
+      box-shadow: rgba(0, 0, 0, .3) 0 0 5px;
+    }
+
+    .content {
+      font-size: 1.5rem;
+      margin-left: 15px;
+    }
+
+    .button {
+      border: none;
+      border-radius: 2px;
+      color: #fafafa;
+      height: 26px;
+      min-width: 32px;
+      padding: 0 16px;
+      margin: 0 16px;
+      text-transform: uppercase;
+      letter-spacing: 0;
+      cursor: pointer;
+      vertical-align: middle;
+      line-height: 26px;
+      text-align: center;
+      background: #3f51b5;
+    }
+
+    header {
+      font-size: 2rem;
+    }
+  </style>
+</head>
+<body>
+
+  <div>
+    <h3>Live updates</h3>
+    <h4>Refresh takes 15 seconds.</h4>
+  </div>
+  <!-- #### amp-live-list with insert -->
+  <!-- 
+    Use `data-poll-interval` attribute to specify how frequently the `amp-live-list` component will poll new data. Values are valid above the minimum of 15 seconds. 
+    Use `data-max-items-per-page` for specifying the maximum number displayed on the page. Additional elements are assumed to be on the next "page".
+    For an extensive explanation of all the attributes, refer to [amp-live-list](https://github.com/ampproject/amphtml/blob/f2aed679a4c0611ea077985cbe0d1926c7d0a6f5/extensions/amp-live-list/amp-live-list.md).
+    
+    Use `on="tap:my-live-list.update"` to load the latest changes from the server. 
+    A button will appear when new items are discovered by the client on polling the server document. Note that the button has to be defined as a direct child of `amp-live-list` .
+
+    A `amp-live-list` must contain a list of items, defined like `<div items>`. Each item has a specific id and the `amp-live-list` will look for items with new id. 
+
+    By clicking on the button "You have updates", new content is displayed on the demo column (the one on the right if the demo is accessed by desktop). The code column (the one in the center if the demo is accessed by desktop) does not update itself, but if you want to see all the items on the page, reload the page.
```

Comment:
put this within `<p class="card">...</p>` to make it explicit that this is something ABE specific.